_includes/e4s-page-actions.html

```diff
@@ -81,7 +81,7 @@
 
   <!-- Open this page in GitHub editor (will prompt to fork if needed) -->
```

Comment:
This comment might need update.  But the gist of it is I think having to fork before you can even see what you would have to edit and what you are getting into is a deterrent.